README.rst

```diff
@@ -104,8 +104,7 @@ like this:
 
 .. code-block:: bash
 
-    pip install -r dev-requirements.txt
-    pytest
+    docker-compose run web tox
```

Comment:
The tests pass when I run this, but I get an error from codecov:
```
INTERNALERROR> AssertionError: must give a directory for html reporting
```
which can be resolved by adding this line to docker-compose.yml under the web 'environment' section:
```
    environment:
      COVERAGE_DIR: htmlcov
```